Project.toml

```diff
@@ -1,25 +1,21 @@
 name = "IPFitting"
 uuid = "3002bd4c-79e4-52ce-b924-91256dde4e52"
-version = "0.4.3"
+version = "0.5.0"
```

Comment:
this should be 0.10.0 please - 0.5.0 is already taken.

Reply:
I propose we use 0.10.x to stabilize this and then decide whether to continue to 0.11.x or 1.x

Reply:
Sounds good to me!